index.html

```diff
@@ -1,85 +1,155 @@
+						in 40 languages relating to 30 categories of business communication.
+						Headquarters in Taipei, Taiwan, it's doing business in China, Hong
+						Kong, Australia and Canada as well. I do part-time
+						Chinese-to-English translation of corporate annual reports for the
+						company.
+					</p>
+					<h3>Code.7370 Trainee</h3>
+					<p>
+						<strong>The Last Mile/CALPIA, September 2016 - August 2018</strong>
+					</p>
+					<p>
+						Sponsored by The Last Mile, the innovative prisoners-to-coders
+						Code.7370 program was launched in 2014 by the California Department
+						of Corrections and Rehabilitation's Prison Industry Authority under
+						its Career Technical Education program. Trainees of the Code.7370
+						program undergo a 18-months training, learning front-end (track 1),
+						back-end (track 2) and advanced algorithm (track 3) in web apps and
+						software development and, once graduated, have the opportunity to
+						work for The Last Mile Works, a for-profit joint venture of TLM and
+						CALPIA that provides work to inmate coders for community projects. I
+						graduated the program while working for CALPIA.
+					</p>
+					<h3>News Writer and Distribution Manager</h3>
+					<p>
+						<strong>San Quentin News, November 2014 - September 2016</strong>
+					</p>
+					<p>
+						The San Quentin News is written and published by the inmates at the
+						San Quentin State Prison with the goal to promote public safety and
+						advance social justice. It is sponsored by multiple grants and
+						individual donations and distributed throughout California prisons
+						and to the community beyond. It won the award of the Society of
+						Professional Journalists for outstanding accomplishments. All of its
+						staff writers are members of the SPJ Northern California chapter,
+						who holds weekly continuing education in prison for the SQN staff. I
+						managed the publication's subscription, distribution, and business
+						analytics and, as a staff writer, spealized in legal news.
+					</p>
+				</seciton>
+
+				<section class="connect-section" id="connect">
+					<h2>Connect</h2>
+					<ul>
+						<li>
+							<a
+								href="mailto:chungckao@gmail.com?subject=About your online portfolio"
+								>Email</a
+							>
+						</li>
+						<li>
+							<a href="https://github.com/Sanlung" target="_blank">GitHub</a>
+						</li>
+						<li>
+							<a href="https://www.linkedin.com/in/chung-kao/" target="_blank"
+								>LinkedIn</a
+							>
+						</li>
+					</ul>
+				</section>
+			</main>
+
+			<footer class="footer">
+				<div class="footer-container">
+					<span>&copy; 2021 by Chung Kao</span>
+					<ul class="social-links">
+						<li class="facebook-logo">Facebook</li>
+						<li class="twitter-log">Twitter</li>
+						<li class="instagram-logo">Instagram</li>
+					</ul>
+				</div>
+			</footer>
+		</div>
+		<script src="js/index.js"></script>
```

Comment:
This is fine for this assignment but Rails has a build it place to put JS, specifically in the `app/javascript/packs` folder and you can reference it in the html using a `<%= javascript_pack_tag '<file_name>', 'data-turbolinks-track': 'reload' %>` tag.  I don't think the class has covered that though so that's purely educational for what's coming.